.github/ISSUE_TEMPLATE/create-new-issues-on-trac.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Create new issues on Trac
+about: Create new issues on Trac
```

Comment:
This appearing twice does seem a bit bizarre, but I guess it doesn't really matter.

![Screenshot_20191111-083835](https://user-images.githubusercontent.com/43215253/68573150-07f32180-045f-11ea-82c5-2b3632e5eae6.jpg)

Reply:
Fixed!